dubbo-config/dubbo-config-api/pom.xml

```diff
@@ -198,25 +192,25 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
```

Comment:
Depedency of configcenter will cause cyclic reference, and related tests needs to move.